Oscar Awards/Logic.java

```diff
@@ -0,0 +1,70 @@
+import java.io.*;
+public class Logic {
+    public static void main(String args[])
+    {
+        try  
+        {  
+            // String path="/home/nikhildewoolkar/Desktop/RedHat_Work/CSV_TO_JSON/the_oscar_award.csv";
+            File file=new File("the_oscar_award.csv");//path
+            FileReader fr=new FileReader(file);
+            BufferedReader br=new BufferedReader(fr);
+            String line;
+            String ans="[";
+            System.out.println("year_film,year_ceremony,ceremony,category,name,film,winner");
+            line=br.readLine();
+            while((line=br.readLine())!=null)  
+            {  
+                String s="[\"";
+                int commacount=0;
+                int i=0;
+                while(i<line.length())
+                {
+                    if(line.charAt(i)!=',' && line.charAt(i)!='\"')
+                    {
+                        s+=line.charAt(i);
+                    }
+                    else if(line.length()>i+1 && line.charAt(i)=='\"')
+                    {
+                        i++;
```

Comment:
I would not recommend modifying the loop index in two different places, that's hard to read and debug. If you need to iterate, please use a for index with clear loop boundaries. Another suggestion is create nested loops

Reply:
I tried doing it using so any logics. It is not possible using String split method because there might be test case like this(1927,1928,1,WRITING (Title Writing),**"George Marion, Jr."**,,False) where string split can make extra clumns because csv can also contain comma in between string represented using "" quotes as shown in example.
I didn't use for loop because this problem cannot be solved without changing index value in between. Using for loop we can't change index. I amcompleting this logic in O(n) complexity usage of for loop will increase time complexity to O(n^2).